py/selenium/webdriver/chromium/options.py

```diff
@@ -121,30 +125,37 @@ def experimental_options(self) -> dict:
     def add_experimental_option(self, name: str, value: Union[str, int, dict, list[str]]) -> None:
         """Adds an experimental option which is passed to chromium.
 
-        :Args:
+        Parameters:
+        ----------
           name: The experimental option name.
           value: The option value.
         """
         self._experimental_options[name] = value
 
     @property
     def enable_webextensions(self) -> bool:
-        """Returns whether webextension support is enabled for Chromium-based browsers.
-
-        :Returns: True if webextension support is enabled, False otherwise.
+        """:Returns: Whether webextension support is enabled for Chromium-based browsers.
+        True if webextension support is enabled, False otherwise.
         """
         return self._enable_webextensions
 
     @enable_webextensions.setter
     def enable_webextensions(self, value: bool) -> None:
         """Enables or disables webextension support for Chromium-based browsers.
 
-        When enabled, this automatically adds the required Chromium flags:
-        - --enable-unsafe-extension-debugging
-        - --remote-debugging-pipe
-
-        :Args:
-         - value: True to enable webextension support, False to disable.
+        Parameters:
+        ----------
+            value : bool
+                True to enable webextension support, False to disable.
+
+        Notes:
+        -----
+        - When enabled, this automatically adds the required Chromium flags:
+            - --enable-unsafe-extension-debugging
+            - --remote-debugging-pipe
+        - Enabling --remote-debugging-pipe makes the connection b/w chromedriver
+        and the browser a pipe instead of a port, disabling many CDP functionalities
```

Comment:
```suggestion
        and the browser use a pipe instead of a port, disabling many CDP functionalities
```
Just a minor nit, else looks good.